tests/PySys/environments/environment_c8y.py

```diff
@@ -10,7 +11,63 @@
 """
 
 
+class Cumulocity(object):
+
+    c8y_url = ""
+    tenant_id = ""
+    username = ""
+    password = ""
+    auth = ""
+
+    def __init__(self, c8y_url, tenant_id, username, password):
+        self.c8y_url = c8y_url
+        self.tenant_id = tenant_id
+        self.username = username
+        self.password = password
+
+        self.auth = ('%s/%s' % (self.tenant_id, self.username), self.password)
+
+    def request(self, method, url_path, **kwargs) -> requests.Response:
+        return requests.request(method, self.c8y_url + url_path, auth=self.auth, **kwargs)
+
+    def getAllDevices(self) -> requests.Response:
```

Comment:
Just a small remark, in python method names are camel_case.

Reply:
Yeah it was kind of a conflict with Pysys which internally uses camelCase instead of snake_case as in `assertThat`, `assertGrep` etc. So, I was also confused. I personally prefer snake_case over camelCase and can stick with that if the team is also aligned on that choice.

Reply:
Sure, I don't  think there is anything to agree to, pep8 is standard python coding style and we have adhered to it before, all tests written in python do adhere to it.
Internal framework code style is not our concern.